### PR TITLE
Add isso integration

### LINKED ISSUE #95: Isso integration
[Isso](https://posativ.org/isso/) is a free and self-hosted disqus alternatives, just use [sphinxnotes-isso](https://sphinx-notes.github.io/isso/) that we can let ablog and isso works together. :)

### PR DESCRIPTION
**Pull request by SilverRainZ**
Close #95.

### CODE REVIEW

**Comment by nabobalis** [comment]
Is this enough to make it work?

**Comment by SilverRainZ** [comment]
On server slide, usershould correctly [configured](https://posativ.org/isso/docs/configuration/server/) their isso service.

On client/sphinx side, it is enough.

File: docs/manual/ablog-configuration-options.rst [hunk 1]
```diff
@@ -244,6 +244,13 @@ The reason for requiring :confval:`blog_baseurl` to be specified as of v0.7.2 is
 
    Choose to disqus posts that are drafts (without a published date), default is ``False``.
 
+Isso integration
+----------------
+
+Simply use `sphinxnotes-isso`_ as other Sphinx document.
+
+.. _sphinxnotes-isso: https://sphinx-notes.github.io/isso/
```
Comment (nabobalis) [inline docs/manual/ablog-configuration-options.rst:252]:
```suggestion
An alternative to Disqus, is `Isso <https://posativ.org/isso/>`__.
Integration is provided by `sphinxnotes-isso`_ and the instructions there. 

.. _sphinxnotes-isso: https://sphinx-notes.github.io/isso/
```

**Review by nabobalis — APPROVED** [review]

**Comment by nabobalis** [comment]
Thanks again @SilverRainZ

**Pull request merged**